README.md

```diff
@@ -1,128 +1,105 @@
+    ```
 3. Start the emulator.
-```
-gt cloud server start
-```
-
-By default, the emulator runs on `127.0.0.1` and port `5000`. You can change these values by providing the `--host` and `--port` options.
-```
-gt cloud server start --host localhost --port 5001
-```
-We'll continue with the default values for the rest of this guide.
-
+    ```bash
+    gt server start
+    ```
 4. In a separate terminal window, register a Structure.
-```
-gt cloud server register --directory skatepark-local --main-file structure.py
-```
-
-5. Run your Structure from a separate program. Here is an example using the `requests` library.
-```python
-import requests
-import time
-
-host = "http://127.0.0.1:5000"
-
-# Start a run with the args "What is 5 ^ 2".
-# These args will be passed into our Structure program as standard input.
-response = requests.post(f"{host}/api/structures/active/runs", json={"env": {}, "args": ["What is 5 ^ 2"]})
-response.raise_for_status()
-
-# Runs are asynchronous, so we need to poll the status until it's no longer running.
-run_id = response.json()["run_id"]
-status = response.json()["status"]
-while status == "RUNNING":
-    response = requests.get(f"{host}/api/runs/{run_id}")
-    response.raise_for_status()
-    status = response.json()["status"]
-    
-    time.sleep(1) # Poll every second.
-
-output = response.json()["output"]
-print(output["value"])
-```
+    ```bash
+    gt server register --directory structure/ --main-file structure.py
+    ```
+
+    The example client program uses the environment variable `GT_STRUCTURE_ID` to determine which Structure to run.
+    Set this environment variable to the Structure ID you registered in the previous step.
+    ```bash
+    export GT_STRUCTURE_ID={STRUCTURE_ID}
+    ```
+
+    Or you can register and set the environment variable in one step.
+    ```bash
+    export GT_STRUCTURE_ID=$(gt server register --directory structure/ --main-file structure.py)
+    ```
+
+> [!IMPORTANT]
+> Structures registered with the emulator are not persisted across restarts. You will need to re-register the Structure each time you restart the emulator.
+5. Confirm that the Structure is registered.
+    ```bash
+    gt server list
+    ```
+6. Add your OPENAI_API_KEY to the `.env` file. You can also add other environment variables here.
+    ```bash
+    echo "OPENAI_API_KEY=sk-XXXXXX-XXXXXX-XXXXXX-XXXXXX" >> structure/.env
+    ```
+7. Rebuild the structure to load in the new environment variable. 
+    Note that this is only required for changes to `.env` or `requirements.txt`. Code changes do not require a rebuild. 
+    ```bash
+    gt server build
+    ```
+8. Now that your Structure is registered, use the example client program to call the emulator's API for running the Structure.
+
+    Navigate to the `example-client` directory.
+    ```bash
+    cd example-client
+    ```
+
+    Install the dependencies required by the example client program.
+    ```bash
+    poetry install
```

Comment:
I thought we did pip?

Reply:
The structure just uses regular pip, the example program uses Poetry.